plugins/livelog/livelog.go

```diff
@@ -90,6 +90,14 @@ func (tp *taskPlugin) Prepare(context *runtime.TaskContext) error {
 }
 
 func (tp *taskPlugin) Finished(success bool) error {
+	return tp.uploadLog(success)
+}
+
+func (tp *taskPlugin) Exception(runtime.ExceptionReason) error {
+	return tp.uploadLog(false)
```

Comment:
Note: adding the boolean, means it won't cause issues if we later ensure that `Finished` is always called.